core/downloadfromdcthread.cpp

```diff
@@ -64,6 +65,27 @@ void DownloadThread::run()
 	dcs->set_device_name(m_data->devBluetoothName());
 }
 
+QList<dc_family_t> ftdiFamilies = {
```

Comment:
Unless this is used in other translation units, you can make it `static const`.

Moreover, `QVector` is usually preferred over `QList`. The latter has some very strange semantics (among other things: what it does depends on the size of the type).

Reply:
right now it's only used here - so yeah, `static const` should be fine.
And QVector vs QList makes no difference to me - all I ever do is use the `.contains()` member as that creates much more readable code than a series of `if(....) ` conditions.

Thanks for the suggestions!